dashboard/app/models/featured_project.rb

```diff
@@ -2,17 +2,16 @@
 #
 # Table name: featured_projects
 #
-#  id                   :integer          not null, primary key
-#  storage_app_id       :integer
-#  who_featured_user_id :integer
-#  featured_at          :datetime
-#  unfeatured_at        :datetime
-#  is_featured          :boolean
+#  id             :integer          not null, primary key
+#  storage_app_id :integer
+#  created_at     :datetime
+#  unfeatured_at  :datetime
 #
 # Indexes
 #
 #  index_featured_projects_on_storage_app_id  (storage_app_id)
 #
 
 class FeaturedProject < ApplicationRecord
+  alias_attribute :featured_at, :created_at
```

Comment:
sorry for not catching this earlier, but it seems like this alias will not give us the right answer in the case where we feature, then un-feature, then re-feature. I would recommend removing this alias and adding a new featured_at column, which we can update to the current time when re-featuring.

Reply:
Good catch. The default timestamps indeed do not work in our case of only caring about the most recent featured time, and possibly an unfeatured time.